servers/zms/scripts/zms_standalone.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+#############
+# Usage: See display_usage
+# Requires zms.war and various system property files and must run as root or sudo user
+# Assumes that this run on AWS.
+###############
+
+if [[ $EUID -ne 0 ]]; then
```

Comment:
@charlesk40 we want this check within the zmsimplmain otherwise I can skip the script and call the zms directly.

Reply:
done